bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
```

Comment:
`ChartOptions` was a model because code was getting written to check values and types, and it seemed reasonable to take advantage of the properties validation capabilities we already have. Perhaps `ChartOptions` should be a `HasProperties` and not `Model`? It doesn't need to be serialized, it's entirely an internal implementation thing on the python side. Alternatively, IIRC it was also created to help with chained methods and as far as I am concerned that can be removed entirely.